### PR TITLE
Os refresh

### PR DESCRIPTION
**Pull request by max-jacobs**
Opening a PR to refresh the README and make some small updates to the Jenkinsfile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
-- Hive metastore events are emitted by the [Hive metastore listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-listener) as a result of the operation.
-- Hive events are picked up from the queue by Beekeeper using [Apiary Receiver](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-receivers).
-- Beekeeper processes these messages and schedules orphaned paths for deletion by adding them to a database.
-- The scheduled paths are deleted by default in 3 days (but can be configured with a different value with table parameter `beekeeper.unreferenced.data.retention.period`).
+| Parameter             | Required | Possible values | Description |
+|:----|:----:|:----:|:----|
+| `beekeeper.remove.unreferenced.data=true`   | Yes |  `true` or `false`       | Set this parameter to ensure Beekeeper monitors your table for orphaned data. |
+| `beekeeper.unreferenced.data.retention.period=X` | No | e.g. `P7D` or `PT3H` (based on [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601)) | Set this parameter to control the delay between schedule and deletion by Beekeeper. Default is 3 days. |
```
Comment (patduin) [inline README.md]: Maybe an example on how to add this parameter to a table using hive?

Comment (max-jacobs) [reply README.md]: Thats a nice idea, will add this

File: README.md [hunk 2]
```diff
@@ -1,20 +1,44 @@
-# Beekeeper
-
 # Overview
-Beekeeper is a service that schedules orphaned paths for cleanup.
+
+Beekeeper is a service that schedules orphaned paths for deletion.
+
+The original inspiration for a data deletion tool came from another of our open source projects called [Circus Train](https://github.com/HotelsDotCom/circus-train). At a high level, Circus Train replicates Hive datasets. The datasets are copied as immutable snapshots to ensure strong consistency and snapshot isolation, only pointing the replicated Hive Metastore to the new snapshot on successful completion. This process leaves behind snapshots of data which are now unreferenced by the Hive Metastore, so Circus Train includes a Housekeeping module to delete these files later.
+
+Circus Train’s Housekeeping module has been re-written and updated to create this service.
```
[the remark this answers is not in the document]
Comment (max-jacobs) [reply README.md]: Yep good point, emphasising that this is decoupled from CT makes sense

File: README.md [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
+
+## Start using
+
+To deploy Beekeeper in AWS, see the [terraform repo](https://github.com/ExpediaGroup/beekeeper-terraform). 
+
+Docker images can be found in Expedia Group's [dockerhub](https://hub.docker.com/u/expediagroup).
```
Comment (abhimanyugupta07) [inline README.md]: can we make this URL specific to Beekeeper. The main URL has other docker images too.

perhaps, https://hub.docker.com/search/?q=expediagroup%2Fbeekeeper

File: README.md [hunk 4]
```diff
@@ -1,20 +1,44 @@
...
+
+## Start using
+
+To deploy Beekeeper in AWS, see the [terraform repo](https://github.com/ExpediaGroup/beekeeper-terraform). 
+
+Docker images can be found in Expedia Group's [dockerhub](https://hub.docker.com/u/expediagroup). 
+
+# How does it work?
+
+Beekeeper makes use of [Apiary](https://github.com/ExpediaGroup/apiary) - an open source federated cloud data lake - to detect changes in the Hive Metastore. One of Apiary’s components, the Apiary Metastore Listener, captures Hive events and publishes these as messages to an SNS topic. Beekeeper uses these messages to detect changes to the Hive Metastore, and perform appropriate deletions.
```
Comment (abhimanyugupta07) [inline README.md]:
```suggestion
Beekeeper makes use of [Apiary](https://github.com/ExpediaGroup/apiary) - an open source federated cloud data lake - to detect changes in the Hive Metastore. One of Apiary’s components, the [Apiary Metastore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/apiary-metastore-listener), captures Hive events and publishes these as messages to an SNS topic. Beekeeper uses these messages to detect changes to the Hive Metastore, and perform appropriate deletions.
```

File: README.md [hunk 5]
```diff
@@ -1,20 +1,44 @@
...
+1. A Hive table is configured with the parameter `beekeeper.remove.unreferenced.data=true` (see [Hive table configuration](#hive-table-configuration) for more details.)
+2. An operation is executed on the table that orphans some data (alter partition, drop partition, etc.)
+3. Hive Metastore events are emitted by the [Hive Metastore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-listener) as a result of the operation.
+4. Hive events are picked up from the queue by Beekeeper using [Apiary Receiver](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-receivers).
+5. Beekeeper processes these messages and schedules orphaned paths for deletion by adding them to a database.
+6. The scheduled paths are deleted by Beekeeper after a configurable delay, the default is 3 days (see [Hive table configuration](#hive-table-configuration) for more details.)
+
+## Hive table configuration
+
+Beekeeper only actions on events which are marked with a specific parameter. This parameter, as well as other configuration parameters, is added to the Hive table that you wish to have Beekeeper monitor. The configuration for tables is as follows:
```
Comment (abhimanyugupta07) [inline README.md]:
```suggestion
Beekeeper only actions on events which are marked with a specific parameter. This parameter, as well as other configuration parameters need to be added to the Hive table that you wish to be monitored by Beekeeper. The configuration parameters for tables are as follows:
```

File: README.md (hunk 3 above)
Comment (max-jacobs) [reply README.md]: looks good, will make that change

**Pull request merged**